jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/utils/ClickAwayListenerTests.java

```diff
@@ -0,0 +1,71 @@
+package io.github.epam.material.tests.utils;
+
+import io.github.epam.TestsInit;
+import javafx.util.Pair;
```

Comment:
need to use another import for Pair, from JDI
import com.epam.jdi.tools.pairs.Pair;

Reply:
done